src/System.Data.OleDb/src/System/Data/ProviderBase/DbParameterHelper.cs

```diff
@@ -0,0 +1,267 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel;
+using System.Data;
+using System.Data.Common;
+using System.Globalization;
+
+namespace System.Data.OleDb {
+    public sealed partial class OleDbParameter : DbParameter { // V1.2.3300
+        private object _value;
+
+        private object _parent;
+
+        private ParameterDirection _direction;
+        private int _size;
+#if USEOFFSET   // USEOFFSET is set in makefile.inc for SqlParameter class only
```

Comment:
`USEOFFSET is set in makefile.inc for SqlParameter class only` -> you probably can delete these blocks then.